lib/pdk/validators/metadata/metadata_syntax.rb

```diff
@@ -11,7 +11,7 @@ def self.name
       end
 
       def self.pattern
-        'metadata.json'
+        '**/metadata.json'
```

Comment:
[PDK-401](https://tickets.puppetlabs.com/browse/PDK-401) specifies _Only validate <module_root>/metadata.json_ so I think this pattern should be unchanged.